trestle/transforms/utils/osco_helper.py

```diff
@@ -217,19 +224,20 @@ def _parse_xml(self) -> Iterator[RuleUse]:
                 weight = lev1.get('weight')
                 result = self._get_result(lev1)
                 rule_use = RuleUse(
-                    id_,
-                    target,
-                    target_type,
-                    benchmark_href,
-                    benchmark_id,
-                    scanner_name,
-                    scanner_version,
-                    idref,
-                    version,
-                    time,
-                    result,
-                    severity,
-                    weight
+                    id_=id_,
```

Comment:
See the comments above. I think a dict might actually make it clearer as the assignment and retrieval can happen at once.

Reply:
Fixed.